mllib/src/main/scala/org/apache/spark/ml/optim/WeightedLeastSquares.scala

```diff
@@ -137,61 +133,68 @@ private[ml] class WeightedLeastSquares(
       }
     }
 
-    // scale aBar to standardized space in-place
-    val aBarValues = aBar.values
-    var j = 0
-    while (j < numFeatures) {
-      if (aStd(j) == 0.0) {
-        aBarValues(j) = 0.0
-      } else {
-        aBarValues(j) /= aStd(j)
-      }
-      j += 1
-    }
+    val bBar = summary.bBar / bStd
```

Comment:
Since we use ```bBar, bbBar, aBar, abBar, aaBar``` in standardized space always, so I did not append ```Std``` as suffix for all variables. If we only add suffix for ```bBar``` and ```bbBar```, developers may misinterpret that other variables are not in standardized space.